2.0/Dockerfile.rhel7

```diff
@@ -14,14 +14,16 @@ LABEL summary="Platform for building and running Ruby 2.0 applications" \
       io.openshift.tags="builder,ruby,ruby20"
 
 # Labels consumed by Red Hat build service
-LABEL com.redhat.component="openshift-sti-ruby-docker" \
-      name="openshift3/ruby-20-rhel7" \
-      version="2.0" \
-      release="2" \
-      architecture="x86_64"
-
-RUN INSTALL_PKGS="ruby200 ruby200-ruby-devel ruby200-rubygem-rake v8314 ror40-rubygem-bundler nodejs010" && \
-  yum install -y --setopt=tsflags=nodocs  $INSTALL_PKGS && rpm -V $INSTALL_PKGS && \
+LABEL BZComponent="openshift-sti-ruby-docker" \
+      Name="openshift3/ruby-20-rhel7" \
+      Version="2.0" \
+      Release="2" \
+      Architecture="x86_64"
+
+RUN yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --enable rhel-7-server-optional-rpms && \
```

Comment:
Well, I might be wrong but as I understand it, it depends on how the host is configured, since I see this on the host:
```
/usr/share/rhel/secrets/rhel7.repo -> /etc/yum.repos.d/redhat.repo
```
So, when there is no rhscl repo enabled on host, which might be on purpose, it won't be enabled during the build either. That is the reason why I think we should enable required repositories explicitly.